x-pack/plugins/actions/common/mustache_template.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/** does the string contain `{{.*}}`? */
+export function hasMustacheTemplate(string: string): boolean {
+  return !!string.match(/.*\{\{.*\}\}.*/);
```

Comment:
A couple suggestions for this mustache template:

1. You don't need to match the whole string (the beginning and the end), you just need to identify if the string _contains_ a mustache template. So you can get rid of the `.*` at the beginning and at the end. Doing that improves performance by an order of magnitude, according to some cursory testing with https://www.myregextester.com/index.php.
2. The `.` token matches any character _except_ line breaks, but technically a mustache template can include line breaks inside of it. So it would be more accurate to match `(.|\n)` instead.

```suggestion
  return !!string.match(/\{\{((.|\n)*)\}\}/);
```

Reply:
good ideas, thx!